useful_workflows/target_workflow_3/target_workflow_3_output_definition.json

```diff
@@ -0,0 +1,171 @@
+[
+  {
+    "input_name": "target_workflow_3.output_file_8",
+    "input_type": {
+      "type": "primitive",
+      "primitive_type": "File"
+    },
+    "source": {
+      "type": "record_lookup",
+      "record_attribute": "target_workflow_3_output_file_8"
```

Comment:
These might need updating as output definitions have `output_name`, `output_type` and `record_attribute` as expected fields.